thrift/src/main/thrift/cluster.thrift

```diff
@@ -114,6 +114,8 @@ struct Node {
   3: required int nodeIdentifier
   4: required int dataPort
   5: required int clientPort
+  // When the cluster is set up for the first time, the clientIp of other nodes is unknown
+  6: required string clientIp
```

Comment:
As clientIP is created, how about change `ip` to another name just like 'internal_ip'?

Reply:
ok